db/seeds.rb

```diff
@@ -9,6 +9,6 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
-5.times do 
+1.times do 
   create(:admin)
```

Comment:
If you want create admin with role: admin, status: active
https://devhints.io/factory_bot -> use with options
<img width="415" alt="Screen Shot 2023-05-04 at 11 00 38" src="https://user-images.githubusercontent.com/115602568/236110069-11bdf43a-ff80-478b-bdca-32f76038dc10.png">
```
create(:admin, role: 'admin', status: 'active')
```